test/speech-credentials.js

```diff
@@ -447,6 +447,28 @@ test('speech credentials tests', async(t) => {
     });
     t.ok(result.statusCode === 204, 'successfully deleted speech credential');
 
+    /* add a credential for nuance */
+    result = await request.post(`/Accounts/${account_sid}/SpeechCredentials`, {
+      resolveWithFullResponse: true,
+      auth: authUser,
+      json: true,
+      body: {
+        vendor: 'cobalt',
+        use_for_stt: true,
+        use_for_tts: false,
+        cobalt_server_uri: 'http://cobalt.com',
```

Comment:
I know this is test data but the uri should be host:port without an http scheme

Reply:
fixed